integration-libs/cds/src/profiletag/services/profiletag-event.service.ts

```diff
@@ -104,7 +104,17 @@ export class ProfileTagEventService {
     this.exposeConfig(newConfig);
   }
 
+  /*
+   * Checks if the script with the given source exists in the document or not.
+   */
+  private isScriptLoaded(scriptSource: string): boolean {
+    return !!this.winRef.document.querySelector(`[src="${scriptSource}"]`);
```

Comment:
Let's add script to it for better readability + maybe one day that script source will be part of another element such as <img src ..> :). Won't really happen, but it would be way more resilent to say to querySelect a script than an attribute with that exact script source only)


```suggestion
    return !!this.winRef.document.querySelector(`script[src="${scriptSource}"]`);
```